cocos/ui/UISlider.h

```diff
@@ -265,6 +265,11 @@ class CC_GUI_DLL Slider : public Widget
      */
     float getZoomScale()const;
 
+    Sprite* getSlidBallNormalRenderer();
+    Sprite* getSlidBallPressedRenderer();
+    Sprite* getSlidBallDisabledRenderer();
+    Node* getSlidBallRenderer();
```

Comment:
I think it is better to add const like
```c++
Sprite* getSlidBallNormalRenderer() const;
```

And why `getSlidBallRenderer ()` return `Node*`, other functions return `Sprite*`?